doc/release-notes.rst

```diff
@@ -265,6 +320,26 @@ Major Changes from Kraken
     - ``ceph tell <daemon> help`` will now return a usage summary.
     - ``ceph fs authorize`` creates a new client key with caps automatically
       set to access the given CephFS file system.
+    - The ``ceph health`` structured output (JSON or XML) no longer contains
+      'timechecks' section describing the time sync status.  This
+      information is now available via the 'ceph time-sync-status'
+      command.
+    - Certain extra fields in the ``ceph health`` structured output that
+      used to appear if the mons were low on disk space (which duplicated
+      the information in the normal health warning messages) are now gone.
+    - The ``ceph -w`` output no longer contains audit log entries by default.
+      Add a ``--watch-channel=audit`` or ``--watch-channel=*`` to see them.
+    - New "ceph -w" behavior - the "ceph -w" output no longer contains
+      I/O rates, available space, pg info, etc. because these are no
+      longer logged to the central log (which is what ``ceph -w``
+      shows). The same information can be obtained by running ``ceph pg
+      stat``; alternatively, I/O rates per pool can be determined using
+      ``ceph osd pool stats``. Although these commands do not
+      self-update like ``ceph -w`` did, they do have the ability to
+      return formatted output by providing a ``--format=<format>``
+      option.
```

Comment:
@theanalyst Now I know what's going on. In Luminous, `ceph -w` just displays the same info as `ceph -s` and then enters an infinite wait loop. It doesn't update anything. I opened http://tracker.ceph.com/issues/20995 to track that bug.

So, please ignore everything I have said up until now, and leave the release note as it is!